index.html

```diff
@@ -263,38 +263,26 @@ <h1>Skills</h1>
             </p>
           </div>
         </div>
-        <div class="contact-form">
+        <div class="contact-form" id="contact-form">
           <form action="https://formspree.io/f/xoqbgdpy">
```

Comment:
- [ ] Good job implementing client side validation, It works and shows an error when I try to submit the form with uppercase letters in the email field, however if we enter a correct email value that is all in lowercase it still is not submitting the form  and sends us to this page :point_down: 
![image](https://user-images.githubusercontent.com/97521557/211068675-56bd504b-40b4-4456-a391-a030ae89ac82.png)

This is because you have not added the method attribute witht value Post to your form element, Please fix this issue :hammer_and_pick: and add a `method="post"` attribute to your form element and make sure the user is able to send the form with correct field data